packages/core/components/AnnotationFilterForm/index.tsx

```diff
@@ -30,17 +41,39 @@ interface AnnotationFilterFormProps {
 export default function AnnotationFilterForm(props: AnnotationFilterFormProps) {
     const dispatch = useDispatch();
     const allFilters = useSelector(selection.selectors.getFileFilters);
+    const fuzzyFilters = useSelector(selection.selectors.getFuzzyFilters);
+    const includeFilters = useSelector(selection.selectors.getIncludeFilters);
+    const excludeFilters = useSelector(selection.selectors.getExcludeFilters);
     const annotationService = useSelector(interaction.selectors.getAnnotationService);
     const [annotationValues, isLoading, errorMessage] = useAnnotationValues(
         props.annotation.name,
         annotationService
     );
 
+    // Check if annotation name is already listed in state as an INCLUDE, EXCLUDE, FUZZY, or regular file filter
+    const usingIncludeFilter: boolean = React.useMemo(() => {
+        return !!includeFilters?.some((filter) => filter.annotationName === props.annotation.name);
+    }, [includeFilters, props.annotation]);
+    const usingExcludeFilter: boolean = React.useMemo(() => {
+        return !!excludeFilters?.some((filter) => filter.annotationName === props.annotation.name);
+    }, [excludeFilters, props.annotation]);
+    const fuzzySearchEnabled: boolean = React.useMemo(() => {
+        return !!fuzzyFilters?.some((filter) => filter.annotationName === props.annotation.name);
+    }, [fuzzyFilters, props.annotation]);
     const filtersForAnnotation = React.useMemo(
         () => allFilters.filter((filter) => filter.name === props.annotation.name),
         [allFilters, props.annotation]
     );
 
+    // Rehydrate filter type from state into UI
+    const defaultFilterType = () => {
```

Comment:
Why is this a function? It looks like we use it only once to pass it to `useState` below (which btw I didn't know you could provide a function to?).